src/Akka.Persistence.Linq2Db.Data.Compatibility.Tests/DataCompatibilityLogicalDeleteSpec.cs

```diff
@@ -0,0 +1,78 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="DataCompatibilityLogicalDeleteSpec.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Configuration;
+using Akka.Hosting;
+using Akka.Persistence.Linq2Db.Data.Compatibility.Tests.Internal;
+using Akka.Persistence.Query;
+using Akka.Persistence.Sql.Compat.Common;
+using Akka.Persistence.Sql.Linq2Db.Query;
+using Akka.Streams;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Persistence.Linq2Db.Data.Compatibility.Tests
+{
+    // Destructive delete with logical delete turned on
+    public abstract class DataCompatibilityLogicalDeleteSpec<T> : DataCompatibilitySpecBase<T> where T : ITestContainer, new()
+    {
+        protected DataCompatibilityLogicalDeleteSpec(ITestOutputHelper output): base(output)
+        {
+        }
+
+        protected override void Setup(AkkaConfigurationBuilder builder, IServiceProvider provider)
+        {
+            builder.AddHocon((Config)@"
+akka.persistence.journal.linq2db.logical-delete = true
```

Comment:
I know there's another open issue for this, but we should not include this option IMHO - a delete is a delete.